locales/ja.json

```diff
@@ -0,0 +1,289 @@
+{
+  "components": {
+    "airport_select": {
+      "placeholder": "空港の選考"
+    },
+    "color_select": {
+      "placeholder": "色彩の選考"
+    },
+    "line_index_select": {
+      "custom_indices": "路線記号の選考",
+      "line_name": "{index}線",
+      "official_indices": "公式の路線記号",
+      "placeholder": "特注の路線記号"
+    },
+    "map_size_select": {
+      "placeholder": "サイズの選考"
+    },
+    "mode_select": {
+      "placeholder": "交通機関の選考"
+    },
+    "service_select": {
+      "placeholder": "交通サービスの選考"
+    },
+    "shape_select": {
+      "placeholder": "形の選考"
+    }
+  },
+  "data": {
+    "airports": {
+      "both": "CDG + ORY",
+      "cdg": "パリ＝シャルル・ド・ゴール空港",
+      "generic": "空港",
```

Comment:
Should not be the word "airport" straight forward, it must give the idea of generic airport or unnamed airport.

Reply:
Since 空港 means simply airport, I think it's understandable that it is any airport. 
But you can use `汎用の空港` for "generic/all-purpose airport" or `無名の空港` for "unnamed airport"

Reply:
After checking the web, reusing just `空港` might be redundant, since it's already in the `空港` menu. So yeah, it's better to add a nuance.